LeaderboardBackend/Filters/PaginatedAttribute.cs

```diff
@@ -52,34 +52,26 @@ public int LimitMax
         set => _limitMax = value;
     }
 
-    public PaginatedAttribute() : base(typeof(PageFilter))
-    {
+    public PaginatedAttribute() : base(typeof(PageFilter)) =>
         Arguments = [new LimitConfigNullable()
         {
             Default = _limitDefault,
             Max = _limitMax
         }];
-    }
 
     private record LimitConfigNullable
     {
         public int? Default { get; init; }
         public int? Max { get; init; }
     }
 
-    private class PageFilter : IAsyncActionFilter
+    private class PageFilter(
+        IOptions<AppConfig> config,
+        LimitConfigNullable limitConfigNullable
+    ) : IAsyncActionFilter
     {
-        private readonly AppConfig _config;
-        private readonly LimitConfigNullable _limitConfigNullable;
-
-        public PageFilter(
-            IOptions<AppConfig> config,
-            LimitConfigNullable limitConfigNullable
-        )
-        {
-            _config = config.Value;
-            _limitConfigNullable = limitConfigNullable;
-        }
+        private readonly AppConfig _config = config.Value;
+        private readonly LimitConfigNullable _limitConfigNullable = limitConfigNullable;
```

Comment:
Don't need these I don't think if using primary constructor. I had problems when using one personally but if you can get it to work then I'll take it.

Reply:
Nah, we don't. I've verified that it works without assigning the primary constructor args to fields, as well.